http/src/main/resources/reference.conf

```diff
@@ -167,10 +167,26 @@ groups {
 
 gke {
   cluster {
-    location = "us-central1-a"
+    location = "us-central1-a",
+    region = "us-central1",
+    authorizedNetworks = ["69.173.127.0/25",
```

Comment:
how are these networks determined?

Reply:
via the list here https://dsp-security.broadinstitute.org/cloud-security/google-cloud-platform/gke. I'll add a comment